docs/getting-started/developers/index.md

```diff
@@ -118,6 +118,10 @@ For more examples how to configure other building tools, please refer to the [ex
 
 #### Add uno.css to your main entry
 
+Depending on the build tool you have chosen this step can look different. The example below is applicable if you build unocss together with vite. If
```

Comment:
It's a very tiny thing but I was wondering, what do you think if we always called the tool `UnoCSS` within Warp docs? I'm mostly thinking about consistency. The same with `Vite` (capital V) and `PostCSS`. Maybe linking to [PostCSS](https://www.npmjs.com/package/postcss) site might also make sense? Wdyt?

Reply:
I agree! I fix